### PR TITLE
Create a reminder email to sign up for peer groups

### LINKED ISSUE #104: Create a reminder email to sign up for peer groups
An email should be sent the last day of the month to everyone who has not yet signed up, reminding them to participate in peer matches.
Once ready, you will need to get in touch with Rebecca @rebeccafae to add the task to Heroku Scheduler.

### PR DESCRIPTION
**Pull request by timkleier**
This commit creates a rake task that, three days before the end of the month, reminds those not yet participating to sign up for peer group matching.

## Note on job scheduling
This rake task can either be added to Heroku Scheduler individually or be lumped in with other rake tasks that run daily, as it has a conditional that will only make it run three days before the end of the month. 

## Github Issue
Fixes #104 

## Screenshot

image

## Definition of Done
- [X] This PR has appropriate test coverage.

### CODE REVIEW

File: spec/lib/tasks/womenrising/peer_group_monthly_matches_rake_spec.rb [hunk 1]
```diff
@@ -1,29 +1,21 @@
 require "rails_helper"
 
-describe "womenrising:peer_group_monthly_match" do
+describe "womenrising:peer_group_signup_reminder" do
```
Comment (rebeccafae) [inline spec/lib/tasks/womenrising/peer_group_monthly_matches_rake_spec.rb]: It looks like the tasks got mixed up based on the names of the files

**Review by rebeccafae — COMMENTED** [review]
Looks great! I'll wait a few days to merge this to give Kate a chance to look at the wording, but it seems good to me!

I added a screenshot, I hope you don't mind

File: spec/lib/tasks/womenrising/peer_group_monthly_matches_rake_spec.rb (hunk 1 above)
Comment (timkleier) [reply spec/lib/tasks/womenrising/peer_group_monthly_matches_rake_spec.rb]: Thanks for catching that, and for adding the screenshot.

**Comment by rebeccafae** [comment]
@timkleier Kate sent me this update for the wording

Remember to sign up for peer groups!
(logo)

It's that time again! We're about to run Women Rising matches for next month and we want YOU to join us!

If you're receiving this email, it's because you haven't opted in for this month yet. To keep everyone engaged we ask you to re-opt-in each month. Just follow the link below (and login if necessary) to participate this month. 
(Participate link)  

While you're at it, why not opt-in to mentoring this month? You never know who YOU could help! 

Cheers,

The Women...

**Review by rebeccafae — APPROVED** [review]

**Comment by rebeccafae** [comment]
This has now been deployed to production. Thanks for all your work @timkleier 😄

**Comment by KateCatlin** [comment]
Thank you both for your work @timkleier @rebeccafae!

**Pull request merged**